docs/source/faqs.md

```diff
@@ -26,6 +27,14 @@ No. The SATRE specification defines a set of stakeholder {ref}`roles <satre_role
 
 We encourage TRE Operators and Builders to publicly evaluate their TREs against the SATRE specification; see {ref}`evaluation`. TRE Developers can use the specification and published TRE evaluations as a starting point. Some of evaluated TREs such as the Alan Turing Institute's "Data Safe Haven" and Dundee HIC's "TREEHOOSE" are deployed from open source infrastructure-as-code, and can be deployed by other institutions.
 
+(support_federation)=
+
+## Does SATRE support federation, interoperability, and related work?
+
+No, however SATRE can form the foundation for future standards and guidance on federation, interoperability, etc.
+
+<!-- If you are interested in getting involved in these discussion please .... contact DARE? A federation WG? Another project? UKTRE mailing list? -->
```

Comment:
Would be good if we could refer people somewhere but not essential

Reply:
Currently, I don't think it is clear who that should be. We could use the SATRE email inbox. Will that be monitored?

Reply:
It will be monitored, but is there anything we can say via that inbox that we can't say here? If not I think we're better off not giving a contact.